modules/tfm/tfm/boards/board/tfm_peripherals_config.h

```diff
@@ -11,18 +11,158 @@
+
+#define TFM_PERIPHERAL_DPPI_SECURE           CONFIG_NRF_DPPI_SECURE
+
+#define TFM_PERIPHERAL_WDT_SECURE            CONFIG_NRF_WDT_SECURE
+
+#define TFM_PERIPHERAL_WDT0_SECURE           CONFIG_NRF_WDT0_SECURE
+
+#define TFM_PERIPHERAL_WDT1_SECURE           CONFIG_NRF_WDT1_SECURE
+
+#define TFM_PERIPHERAL_COMP_SECURE           CONFIG_NRF_COMP_SECURE
+
+#define TFM_PERIPHERAL_LPCOMP_SECURE         CONFIG_NRF_LPCOMP_SECURE
+
+#define TFM_PERIPHERAL_EGU0_SECURE           CONFIG_NRF_EGU0_SECURE
+
+#define TFM_PERIPHERAL_EGU1_SECURE           CONFIG_NRF_EGU1_SECURE
+
+#define TFM_PERIPHERAL_EGU2_SECURE           CONFIG_NRF_EGU2_SECURE
+
+#define TFM_PERIPHERAL_EGU3_SECURE           CONFIG_NRF_EGU3_SECURE
+
+#define TFM_PERIPHERAL_EGU4_SECURE           CONFIG_NRF_EGU4_SECURE
+
+#define TFM_PERIPHERAL_EGU5_SECURE           CONFIG_NRF_EGU5_SECURE
+
+#define TFM_PERIPHERAL_PWM0_SECURE           CONFIG_NRF_PWM0_SECURE
+
+#define TFM_PERIPHERAL_PWM1_SECURE           CONFIG_NRF_PWM1_SECURE
+
+#define TFM_PERIPHERAL_PWM2_SECURE           CONFIG_NRF_PWM2_SECURE
+
+#define TFM_PERIPHERAL_PWM3_SECURE           CONFIG_NRF_PWM3_SECURE
+
+#ifdef NRF_PDM0
+#define TFM_PERIPHERAL_PDM0_SECURE           CONFIG_NRF_PDM_SECURE
+#else
+#define TFM_PERIPHERAL_PDM_SECURE            CONFIG_NRF_PDM_SECURE
 #endif
 
-#if TEST_NS_SLIH_IRQ || TEST_NS_FLIH_IRQ
-#define TFM_PERIPHERAL_TIMER0_SECURE 1
+#ifdef NRF_I2S0
+#define TFM_PERIPHERAL_I2S0_SECURE           CONFIG_NRF_I2S_SECURE
+#else
+#define TFM_PERIPHERAL_I2S_SECURE            CONFIG_NRF_I2S_SECURE
 #endif
 
-#ifdef PSA_API_TEST_IPC
-#define TFM_PERIPHERAL_EGU5_SECURE 1
+#define TFM_PERIPHERAL_IPC_SECURE            CONFIG_NRF_IPC_SECURE
+
+#define TFM_PERIPHERAL_QSPI_SECURE           CONFIG_NRF_QSPI_SECURE
+
+#define TFM_PERIPHERAL_NFCT_SECURE           CONFIG_NRF_NFCT_SECURE
+
+#define TFM_PERIPHERAL_MUTEX_SECURE          CONFIG_NRF_MUTEX_SECURE
+
+#define TFM_PERIPHERAL_QDEC0_SECURE          CONFIG_NRF_QDEC0_SECURE
+
+#define TFM_PERIPHERAL_QDEC1_SECURE          CONFIG_NRF_QDEC1_SECURE
+
+#define TFM_PERIPHERAL_USBD_SECURE           CONFIG_NRF_USBD_SECURE
+
+#define TFM_PERIPHERAL_USBREG_SECURE         CONFIG_NRF_USBREG_SECURE
+
+#define TFM_PERIPHERAL_NVMC_SECURE           CONFIG_NRF_NVMC_SECURE
+
+#define TFM_PERIPHERAL_GPIOTE0_SECURE        CONFIG_NRF_GPIOTE0_SECURE
+
+#define TFM_PERIPHERAL_GPIO0_SECURE          CONFIG_NRF_GPIO0_SECURE
+
+#define TFM_PERIPHERAL_GPIO1_SECURE          CONFIG_NRF_GPIO1_SECURE
+
+#define TFM_PERIPHERAL_VMC_SECURE            CONFIG_NRF_VMC_SECURE
+
+#if defined(CONFIG_NRF_GPIO1_PIN_MASK_SECURE)
+#define TFM_PERIPHERAL_GPIO0_PIN_MASK_SECURE CONFIG_NRF_GPIO0_PIN_MASK_SECURE
+#endif
+
+#if defined(CONFIG_NRF_GPIO1_PIN_MASK_SECURE)
+#define TFM_PERIPHERAL_GPIO1_PIN_MASK_SECURE CONFIG_NRF_GPIO1_PIN_MASK_SECURE
+#endif
 
-#define TFM_PERIPHERAL_WDT0_SECURE 1
+/* TODO: Update to TFM_PERIPHERAL_DPPI_CHANNEL_SECURE_MASK once this is renamed in TF-M. */
```

Comment:
Rename what to that? Isn't it already called that here?

Reply:
Comment is wrong, should be `TFM_PERIPHERAL_DPPI_CHANNEL_MASK_SECURE`